src/ledger/AccountFrame.cpp

```diff
@@ -118,10 +118,82 @@ AccountFrame::getBalance() const
     return (mAccountEntry.balance);
 }
 
+int64_t
+getBuyingLiabilities(AccountEntry const& acc, LedgerManager const& lm)
+{
+    assert(lm.getCurrentLedgerVersion() >= 10);
```

Comment:
We haven't followed this convention with other version numbers, and I'm not sure it would add much clarity. For example, do you think `assert(lm.getCurrentLedgerVersion() >= PROTOCOL_VERSION_10);` would be considerably more clear?

Reply:
The reason I suggested it:
1. You can easily change it. Previously you had 11, so instead of changing it to 10 in every place, you could just change one variable. 
2. you could name it something like `PROTOCOL_VERSION_WITH_LIABILITIES` and it would be way less vague than just `10`

Reply:
let's not do that in this PR: v10 is broader than liabilities.

Reply:
yeah like I said, not for this PR; and this would make the code even more complicated to understand as features depend on each other on a per protocol basis.